tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -325,6 +334,9 @@ export class CardFobControllerComponent {
    * fob toggles the feature entirely.
    */
   onFobRemoved(fob: Fob) {
+    if (!this.timeSelection) {
+      return;
+    }
```

Comment:
nit: If there is no timeSelection, there is no fob, and this method should not be triggered. Can we do `this.timeSelection!` in this method or you prefer a check and return?
Same as `stepTyped` method.

Reply:
I really don't like adding non null assertions but it does make sense in this case.